packages/barcode-scanning/android/src/main/java/io/capawesome/capacitorjs/plugins/mlkit/barcodescanning/BarcodeScannerHelper.java

```diff
@@ -30,7 +30,14 @@ public static JSObject createBarcodeResultForBarcode(@NonNull Barcode barcode, @
                 cornerPointsResult.put(cornerPointResult);
             }
         }
-
+        else if (cornerPoints != null) {
```

Comment:
This also affects the `scan(...)` method and leads to invalid values there, as these would have to be normalized.
For now, my suggestion is to make this dependent on the `screenSize` parameter. If this parameter is set, corner points may only be returned normalized. I will improve this later.

```suggestion
        else if (cornerPoints != null && screenSize == null) {
```

Make sure that the `screenSize` parameter is `null` only for the `readBarcodesFromImage(...)` method.
Please also apply this to iOS.